OpenSky_Get_Data.py

```diff
@@ -71,39 +61,36 @@ def getter(init_time, bounds, timer, anam):
                                   bounds=bounds,
                                   other_params=" and time-lastcontact<=15 ")
         flights.to_pickle(outf)
+        print("Wrote", outf)
     except Exception as e:
         print("There is a problem with this date/time combination:", e, times)
 
     return
 
 
-bounds = get_bounds(AIRPRT.rwy_list)
-
-# Loop over timestamps to retrieve all the data.
-while True:
-
-    dtst = start_dt.strftime("%Y%m%d%H%M")
-    outf = outdir + 'OS_' + dtst+'_' + AIRPRT.icao_name + '.pkl'
-    print("Now processing:",
-          start_dt.strftime("%Y/%m/%d %H:%M"),
-          'for', AIRPRT.airport_name + ' / ' +
-          AIRPRT.icao_name)
-
-    # Create processes for each hour, in total 'nummer' hours are
-    # processed simultaneously
-    processes = [mp.Process(target=getter,
-                            args=(start_dt, bounds, i, AIRPRT.icao_name))
-                 for i in range(0, nummer)]
-
-    # Start, and then join, all processes
-    for p in processes:
-        p.start()
-    for p in processes:
-        p.join()
-
-    # Move on to the next block of times
-    start_dt = start_dt + timedelta(hours=nummer)
-
-    # If we have reached the end of the block then exit
-    if (start_dt >= end_dt):
-        break
+@click.command()
+@click.option('--airport', default='VABB')
+@click.option('--start-dt', default='2019-08-10')
+@click.option('--end-dt', default='2019-08-21')
+@click.option('--outdir', default='INDATA/')
+@click.option('--n-jobs', default=1)
+def main(airport, start_dt, end_dt, outdir, n_jobs):
+    os.makedirs(outdir, exist_ok=True)
+    airport = import_module('OS_Airports.' + airport)
+    bounds = get_bounds(airport.rwy_list)
+    start_dt = datetime.strptime(start_dt, '%Y-%m-%d').replace(
+        tzinfo=timezone.utc)
+    end_dt = datetime.strptime(end_dt, '%Y-%m-%d').replace(
+        tzinfo=timezone.utc)
+    hours = int((end_dt - start_dt).total_seconds() / 60 / 60 + 0.5)
+
+    pool = mp.Pool(n_jobs)
+
+    pool.starmap(getter, [
+        (start_dt, bounds, hour, airport.icao_name, outdir)
+        for hour in range(hours)
+        ])
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
It looks like this section substantially changes the way the code runs (sorry, I can't actually test it right now).
Does this affect the original functionality of the code? i.e: Can I still run it the same way as before? Or does it now run a different way, from the command line?

Reply:
Without any arguments, it runs just the same as before, with the exception of the output directory, which now defaults to `INDATA` in the current working directory.  See my additions to README.

Reply:
Ok, thanks.